docs/plans/2026-04-18-brainbar-dashboard-flow-collab.md

```
@@ -0,0 +1,38 @@
+# BrainBar Dashboard Flow Collab
+
+## Goal
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)